tox.ini

```diff
@@ -11,25 +11,24 @@ skipsdist = True
 extras = dev
 
 [testenv:test]
-deps = -r operator-pipeline-images/requirements-dev.txt
-       -r operator-pipeline-images/requirements.txt
-
+groups = operatorcert
+         operatorcert-dev
 commands = pytest -v \
        --cov {[vars]OPERATOR_MODULE} \
        --cov-report term-missing \
        --cov-fail-under 100
 
 [testenv:black]
-deps = -r operator-pipeline-images/requirements-dev.txt
+groups = operatorcert-dev
 commands = black --check --diff .
 
 [testenv:black-format]
-deps = -r operator-pipeline-images/requirements-dev.txt
+groups = operatorcert
```

Comment:
Shouldn't this be `operatorcert-dev` instead?

Reply:
Updated.